app/src/main/kotlin/com/wire/android/ui/home/conversations/search/messages/SearchConversationMessagesResultsScreen.kt

```diff
@@ -64,17 +85,3 @@ fun SearchConversationMessagesResultsScreen(
         }
     }
 }
-
-@PreviewMultipleThemes
-@Composable
-fun previewSearchConversationMessagesResultsScreen() {
-    WireTheme {
-        SearchConversationMessagesResultsScreen(
-            searchResult = listOf(
-                mockMessageWithText,
-                mockMessageWithText,
-            ),
-            onMessageClick = {}
-        )
-    }
-}
```

Comment:
Removed preview as we can't pass `LazyPagingItems` with dummy items

Reply:
@alexandreferris we can 😄 
just use `flowOf(PagingData.from(listOf())).collectAsLazyPagingItems()`
but if you want to use mock messages, then you need to make sure each one has unique id:
```
lazyPagingMessages = flowOf(
                PagingData.from(
                    listOf<UIMessage>(
                        mockMessageWithText.copy(header = mockMessageWithText.header.copy(messageId = "1")),
                        mockMessageWithText.copy(header = mockMessageWithText.header.copy(messageId = "2")),
                    )
                )
            ).collectAsLazyPagingItems(),
```